File: php/fib.php
<?php
function fibonacci($n) {
  return ($n <= 1) ? $n : (fibonacci($n - 1) + fibonacci($n - 2));
}

try {
  $sequences = intval($argv[1]);
  $start = hrtime(true);
  $output = fibonacci($sequences);
  $end = hrtime(true);
  $elapsed = intval(round(($end - $start) / 1000000));
  echo "PHP Fib: {$sequences}x\nResult: {$output} | Time: {$elapsed}ms\n";
} catch (Throwable $exception) {
  echo "{$exception}\n";
}

File: java/fib.java
public class fib {
  static long fibonacci(int n) {
    return (n <= 1) ? n : fibonacci(n - 1) + fibonacci(n - 2);
  }

  public static void main(String[] args) {
    try {
      int sequences = Integer.parseInt(args[0]);
      long start = System.currentTimeMillis();
      long output = fib.fibonacci(sequences);
      long finish = System.currentTimeMillis();
      long elapsed = (finish - start);
      System.out.printf("Java Fib: %dx\nResult: %d | Time: %dms\n", sequences, output, elapsed);
    } catch (Exception e) {
      System.out.println(e);
    }
  }
}